Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1474,8 +1512,11 @@ internal async static Task CopyObject_Test8(MinioClient minio)
                 };
                 await minio.CopyObjectAsync(bucketName, objectName, destBucketName, destObjectName, copyConditions:copyCond, metadata: metadata);
 
-                ObjectStat dstats = await minio.StatObjectAsync(destBucketName, destObjectName);
-                Assert.IsTrue(dstats.MetaData["Mynewkey"] != null);
+                statObjectArgs = new StatObjectArgs()
+                                            .WithBucket(destBucketName)
+                                            .WithObject(destObjectName);
+                ObjectStat dstats = await minio.StatObjectAsync(statObjectArgs);
+                Assert.IsTrue(dstats.MetaData["mynewkey"] != null);
```

Comment:
Again, here it is specifically checking that metadata key is case sensitive

Reply:
Metadata dictionary with comparer IgnoreCase. Code updated.